.gitlab-ci.yml

```diff
@@ -30,7 +30,7 @@ variables:
   RUSTY_CACHIER_COMPRESSION_METHOD: zstd
   NEXTEST_FAILURE_OUTPUT: immediate-final
   NEXTEST_SUCCESS_OUTPUT: final
-  ZOMBIENET_IMAGE: "docker.io/paritytech/zombienet:v1.3.65"
+  ZOMBIENET_IMAGE: "docker.io/paritypr/zombienet:8c572276"
```

Comment:
> This will be changed to the next version?

```suggestion
  ZOMBIENET_IMAGE: "docker.io/paritypr/zombienet:v1.3.66"
```
The needed changes are already part of the latest release 👍